src/coreclr/pal/inc/pal.h

```diff
@@ -4334,4 +4334,10 @@ class NativeExceptionHolderFactory
 }
 #endif
 
+#ifndef TARGET_WASM
```

Comment:
Nit: consider standardizing it with the existing (more general) pattern we have used in this file (and elsewhere):
```suggestion
#if __has_builtin(__builtin_return_address)
```

Reply:
These two conditions are not equivalent. Emscripten has `__builtin_return_address`, it just isn't (and can't be) implemented in a useful way.